nailgun/entities.py

```diff
@@ -1124,6 +1128,23 @@ def promote(self, synchronous=True, **kwargs):
         response = client.post(self.path('promote'), **kwargs)
         return _handle_response(response, self._server_config, synchronous)
 
+    def inc_update(self, synchronous=True, **kwargs):
```

Comment:
Please add some test coverage for this method. You can just copy and paste the relevant bits from https://github.com/SatelliteQE/nailgun/compare/master...Ichimonji10:inc-update